test/index.test.js

```diff
@@ -0,0 +1,39 @@
+const { expect } = require('chai')
+const { describe, it, after, before } = require('mocha')
+const request = require('request')
+const createApp = require('../create-app')
+require('dotenv').config()
+
+describe('Express Server', () => {
+  const app = createApp()
+  let server
+  before(done => {
+    server = app.listen(process.env.PORT, () => {
+      done()
+    })
+  })
+
+  after(done => {
+    server.close(() => {
+      done()
+    })
+  })
+  describe('GET Request', () => {
+    it('should get the information of name and description of repo', async () => {
+      request('http://localhost:3000/', (err, response, body) => {
```

Comment:
`request` takes a callback, so it's better to call `done` when the callback is run, instead of supplying an `async` test.